tests/test_routes.py

```diff
@@ -42,16 +43,42 @@ def setUp(self):
         db.session.commit()
 
     def tearDown(self):
-        """ This runs after each test """
+        """This runs after each test"""
         db.session.remove()
 
     ######################################################################
     #  P L A C E   T E S T   C A S E S   H E R E
     ######################################################################
 
     def test_index(self):
-        """ It should call the home page """
-        resp = self.client.get("/")
-        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        """It should call the home page and receive information about existing methods"""
+        response = self.client.get("/")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        methods = data["methods"]
+        self.assertEqual(data["status"], status.HTTP_200_OK)
+        self.assertEqual(data["name"], "Payments service")
+        self.assertEqual(data["version"], "1.0")
+        self.assertEqual(len(methods), 5)
+
+        # check if root path has definitions for all methods
+        def is_path_and_method_in_list(path, method):
```

Comment:
I decided not to test the description as it can change frequently. I think it's just important to test if all appropriate methods are defined.